blogposts/2021/release-3.34

```diff
@@ -0,0 +1,26 @@
+---
+title: "Sourcegraph 3.34 release"
+publishDate: 2021-11-23T10:00-07:00
+description: "Sourcegraph 3.34 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.34"
+published: false
+heroImage:
+socialImage:
+changelogItems:
+
+---
+
+Sourcegraph 3.34 is now available! For this release, we introduced:
+
+## Server side Batch Changes (Experimental)
+
+By default, Batch Changes uses a command line interface in your local environment to compute batch changes. This can be impractical for creating batch changes affecting hundreds or thousands of repositories or if the batch change steps require CPU, memory, or disk resources that are not locally.
```

Comment:
```suggestion
By default, Batch Changes uses a command line interface in your local environment to compute batch changes. This can be impractical for creating batch changes affecting hundreds or thousands of repositories or if the batch change steps require CPU, memory, or disk resources that are not available locally.
```